chadwick_bissette/lib/greet.js

```diff
@@ -0,0 +1,5 @@
+'use strict';
+
+module.exports =  function(name) {
+  return 'hello ' + name;
+}
```

Comment:
This is nitpicking, but always put an empty trailing line at the end of your files.